fxa_refreshing/provider.py

```diff
@@ -0,0 +1,61 @@
+import requests
+
+from allauth.compat import parse_qsl
+from allauth.socialaccount.providers.oauth2.client import (
+    OAuth2Client, OAuth2Error
+)
+from allauth.socialaccount.providers.fxa.provider import FirefoxAccountsProvider
+
+
+class RefreshingFXAClient(OAuth2Client):
+    # Custom client to include access_type=offline and store refresh token
```

Comment:
I thought you can do request a refresh token like this:
```
SOCIALACCOUNT_PROVIDERS = {
    'google': {
        'SCOPE': [
            'profile',
            'email',
        ],
        'AUTH_PARAMS': {
            'access_type': 'online',
        }
    }
}
```
Like described in https://django-allauth.readthedocs.io/en/latest/providers.html#django-configuration

Reply:
Can you clarify whta's different to the default OAuth2Client?

Reply:
holy crap yes this is what I needed the whole time! Closing this PR to open a dramatically more simple one.